### PR TITLE
FIX: Not actually rendering the inline hints

### PR DESCRIPTION
**Pull request by adrianbrowning**
FIX: Not actually rendering the inline hints
REFACTOR: Bring up-to-date with the latest template

### CODE REVIEW

File: src/test/kotlin/yij/ie/ideacommentqueries/providers/CommentCollectorKtTest.kt [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
+package yij.ie.ideacommentqueries.providers
 
-import org.junit.Test
-import yij.ie.ideacommentqueries.providers.defineAbsoluteMatcher
-import yij.ie.ideacommentqueries.providers.defineRelativeMatcherRegExp
-import yij.ie.ideacommentqueries.providers.resolveRelativeMatchResult
-import kotlin.test.assertEquals
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+
+internal class CommentCollectorKtTest  : BasePlatformTestCase() {
```
Comment (NWYLZW) [inline src/test/kotlin/yij/ie/ideacommentqueries/providers/CommentCollectorKtTest.kt]:
```suggestion
internal class CommentCollectorKtTest : BasePlatformTestCase() {
```

Looks like a coding style error?

**Review by NWYLZW — APPROVED** [review]
LGTM, thank you for the work you do

**Comment by adrianbrowning** [comment]
@NWYLZW don't suppose you could do a build, and push it to the marketplace. 
Many thanks!

**Comment by NWYLZW** [comment]
> @NWYLZW don't suppose you could do a build, and push it to the marketplace. Many thanks!

Yes, I will test it locally, but I will not release a new version for the time being. I will try to add a configuration item to control the turning on and off of the function.

**Pull request merged**